docs/README.md

```diff
@@ -48,7 +48,7 @@ Bireylerin internet ortamında daha güvenli bir deneyim yaşamalarını sağlam
     <img src="docs/images/ublock.png" alt="Logo">
     <div style="flex: 1;">
         <a href="https://guvendekal.org/#/tarayici-eklentileri?id=-ublock-origin" class="title">En iyi reklam ve izleyici engelleyici: uBlock Origin</a>
-        <p class="description">En güçlü ve gelişmiş reklam, izleyici engelleme eklentisidir. Diğer reklam engelleme eklentilerine kıyasla daha gelişmiş ve reklam engelleme yeteneği daha iyidir.</p>
+        <p class="description">Sıradan bir reklam engelleyici değil, işlemci ve bellek verimliliğine odaklanan güçlü bir içerik engelleyicidir.</p>
```

Comment:
İçerik engelleyicisidir tanımı biraz garip kalıyor gibi farklı bir açıklama kullanılabilir mi acaba?

Reply:
Açıkçası bana da biraz tuhaf geldi ama uBlock Origin'in kendini tanımlama şekli bu. Reklam engelleyici olduğunu da belirtmek için başta "sıradan bir reklam engelleyici değil" gibi bir ifade kullandım. Böylece ne kadar "güçlü" olduğunu anlayabilirler.

Reply:
içerik derken kast ettiği "reklam, çerez ve izleyici" gibi 3. parti şeyler büyük ihtimalle. Belki belirtirsek daha bütünsel durabilir belki.

Reply:
uBlock Origin kelimenin tam anlamıyla herhangi bir "içeriği" engellemeye imkan tanıyor. Örneğin YouTube sol menü çubuğundaki "Shorts" sekmesini uBlock Origin ile tamamen kaldırabilirsiniz. Ya da herhangi bir sitede dikkatinizi dağıtan herhangi bir element varsa bunu yine bu eklentiyle tamamen yok edebilirsiniz.